fastlane/lib/fastlane/actions/cocoapods.rb

```diff
@@ -15,7 +15,9 @@ def self.run(params)
         end
 
         cmd << ['bundle exec'] if params[:use_bundle_exec] && shell_out_should_use_bundle_exec?
-        cmd << ['pod install']
+        cmd << ['pod']
+        cmd << [" _#{params[:pod_version]}_"] if params[:pod_version]
```

Comment:
Alright, thanks! Note that you were also adding spaces around this and `install` later which shouldn't be needed since we're joining all params together on line 29 anyway. If you remove them, the tests should pass 👍